styles/twitch/catppuccin.user.css

```diff
@@ -590,6 +590,8 @@
       --color-text-label: @text;
       --color-border-input-hover: @surface0;
       --color-opac-b-14: @crust;
+      --color-text-button-overlay-primary: @text !important;
```

Comment:
That doesn't seem to be working, apparently the right variable is `--color-text-button-overlay` and not `--color-text-button-overlay-primary`.

Reply:
Also the volume slider isn't affecter by this var, that requires adding `--color-background-range-overlay-fill: @text !important;` too.

Reply:
fair, I did not check it before merging, now I applied what you mentioned and did a few edits, I think this looks good
![image](https://github.com/catppuccin/userstyles/assets/42213155/5a470ba5-958b-4dd9-92a6-a2ae9b17159d)